packages/cli/src/util/link/setup-and-link.ts

```diff
@@ -128,6 +130,20 @@ export default async function setupAndLink(
   } else {
     const project = projectOrNewProjectName;
 
+    const remoteUrls = await getRemoteUrls(join(path, '.git/config'), output);
+    if (remoteUrls && !project.skipGitConnectDuringLink) {
+      const connectGit = await handleGitConnection(
+        client,
+        org,
+        output,
```

Comment:
Since `output` and `client.output` are the same, and the code is already passing `client` around, maybe remove `output` and pull `output` out of `client` when needed?

Reply:
Thanks, great suggestion.